src/config/mod.rs

```diff
@@ -103,6 +130,17 @@ impl Config {
             None => Err(ConfigError::UnsupportedCiphersuite),
         }
     }
+
+    /// Get the default `KeyPackage` lifetime (in seconds).
+    pub fn default_key_package_lifetime() -> &'static u64 {
```

Comment:
For primitives we should just return the value.

Reply:
I was wondering about that. Will do.